### PR TITLE
Add GitLab integration section

### PR DESCRIPTION
**Pull request by dadlerj**
Adds a new section to the homepage, after customer logos

image

### CODE REVIEW

File: website/src/components/product/GitLabIntegrationSection.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    <div className="gitlab-integration-section py-5 row justify-content-center ">
+        <div className="row justify-content-center container">
+            <h2 className="text-center display-4 mb-2">
+                GitLab adds code navigation with Sourcegraph
+            </h2>
+            <div className="col-lg-8 mb-6 mb-lg-0">
+                <div className=" gitlab-integration-section__quote">
+                    <p className="font-weight-light">
+                        Sourcegraph has the best find-definition, find-references, and intelligent code navigation capability on the planet - and they brought it to GitLab.
```
Comment (sqs) [inline website/src/components/product/GitLabIntegrationSection.tsx]:
```suggestion
                        Sourcegraph has the best find-definition, find-references, and intelligent code navigation capability on the planet&mdash;and they brought it to GitLab.
```

File: website/src/components/product/GitLabIntegrationSection.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { RequestDemoAction } from '../../css/components/actions/RequestDemoAction'
+
+export const GitLabIntegrationSection: React.FunctionComponent<{ className?: string }> = ({
+    className = '',
+}) => (
+
+    <div className="gitlab-integration-section py-5 row justify-content-center ">
+        <div className="row justify-content-center container">
+            <h2 className="text-center display-4 mb-2">
+                GitLab adds code navigation with Sourcegraph
```
Comment (sqs) [inline website/src/components/product/GitLabIntegrationSection.tsx]: Some kind of "New" badge would be good here - like:
image

Comment (ryan-blunden) [reply website/src/components/product/GitLabIntegrationSection.tsx]: @dadlerj The styles for the `private beta` on the automation page may help.

**Comment by dadlerj** [comment]
Updated 

image

File: website/src/components/product/GitLabIntegrationSection.tsx (hunk 2 above)
Comment (dadlerj) [reply website/src/components/product/GitLabIntegrationSection.tsx]: Yep that's exactly what I pulled from!

**Pull request merged**